openedx/features/caliper_tracking/transformers/enrollment_transformers.py

```diff
@@ -0,0 +1,15 @@
+"""
+Transformers for all course related events.
+"""
+
+
+def edx_course_enrollment_activated(current_event, caliper_event):
+    caliper_event.update({
+        'ip': current_event['ip'],
+        'object': current_event['event'],
```

Comment:
@tasawernawaz @mumarkhan999 don't we need `object` as a dictionary?

Reply:
I assume `current_event['event']` will return dictionary.